base/lterror.dtx

```diff
@@ -375,6 +375,58 @@ Type  H <return>  for immediate help.}%
 % \end{macro}
 % \end{macro}
 %
+%
+%
+%  \begin{macro}{\ClassNote,\ClassNoteNoLine,
+%                \PackageNote,\PackageNoteNoLine}
+%    
+% \changes{v1.2s}{2021/07/11}{Provide \cs{ClassNote} and \cs{PackageNote}}
+%    \begin{macrocode}
+%</2ekernel>
+%<*2ekernel|latexrelease>
+%<latexrelease>\IncludeInRelease{2021/11/15}%
+%<latexrelease>                 {\ClassNote}{Notes for classes/packages}%
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\def\ClassNote#1#2{%
+   \GenericWarning{%
+      (#1) \space\space\@spaces\@spaces
+   }{%
+      Class #1 Info: #2%
```

Comment:
Shouldn't this be `Note` instead of `Info`?

Reply:
We could have a different word here. But my thought was it is just information one goes to the terminal+log the other just to the "log". Also not that at the moment the L3 interface also writes info (I think) in both cases

Reply:
I thought of using `Note` instead because of the name of the command.  But indeed, `expl3`'s `\msg_note:nnn` says `Info`, so nevermind this comment

Reply:
I think it is ok to use the same string here.